src/frontend/packages/lib_video/src/components/live/common/Chat/ChatMessageGroupItem/index.tsx

```diff
@@ -33,7 +33,13 @@ export const ChatMessageGroupItem = ({
         </Box>
       </Box>
     ),
-    [firstMessage.sentAt, msgGroup.sender, msgGroup.messages],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [
+      firstMessage.sentAt,
+      msgGroup.sender,
+      msgGroup.messages,
+      msgGroup.messages.length,
```

Comment:
`msgGroup.messages.length` is more a trick. 
The root of the prob is [here](https://github.com/openfun/marsha/blob/master/src/frontend/packages/lib_video/src/hooks/useChatItemsStore/index.ts#L64), because of the shallow copy the useMemo doesn't trigger, the reference is the same.

2 solutions:
 - Solution 1: use `return cloneDeep({ chatItems: [...state.chatItems] });` [here](https://github.com/openfun/marsha/blob/master/src/frontend/packages/lib_video/src/hooks/useChatItemsStore/index.ts#L64), `cloneDeep` comes from `lodash`.
  - Solution 2: Don't use the `useMemo` [here](https://github.com/openfun/marsha/blob/master/src/frontend/packages/lib_video/src/components/live/common/Chat/ChatMessageGroupItem/index.tsx#L17) and just write a normal component.

Personally I would choose solution 2, at first sight, I don't see the point to use `useMemo` in this situation.